src/tests/Microsoft.PowerFx.Connectors.Tests/BasicRestTests.cs

```diff
@@ -137,5 +142,165 @@ public async Task PetStore_MultiServer()
             // Ensure we use HTTPS protocol
             Assert.Equal("https", apiDoc.GetScheme().Substring(0, 5));
         }
+
+        [Fact]
+        public async Task RealTest()
+        {
+            var config = new PowerFxConfig();
+            
+            // Note that these APIs are rate limited and HTTP error 429 is possible
+            var swaggerUrl = "https://api.math.tools/yaml/math.tools.numbers.openapi.yaml";
+
+            // Other intersting files:
+            // "https://api.apis.guru/v2/specs/weatherbit.io/2.0.0/swagger.json"
+            // "https://www.weatherbit.io/static/swagger.json"
+
+            OpenApiDocument doc = await ReadSwaggerFromUrl(swaggerUrl).ConfigureAwait(false);
+
+            // No BaseAdress specified, we'll use the 1st one found in the swagger file
+            using var client = new HttpClient(); // public auth             
+            var funcs = config.AddService("Math", doc, client);
+
+            var engine = new RecalcEngine(config);
+            var expr = "Math.numberscardinal({number: 1791941})";
```

Comment:
>var expr = "Math.numberscardinal({number: 1791941})"; [](http://example.com/codeflow?start=12&length=53)

Can't access return result:
Math.numberscardinal({number: 1791941}).contents.cardinal

    [0]: {Error 39-48: Name isn't valid. 'contents' isn't recognized.}
    [1]: {Error 48-57: Invalid use of '.'}